### ISSUE TITLE
Other Pretrained Models with Seq-Labeling Task.

### ISSUE THREAD

**Issue by SibtainRazaJamali** [microsoft/unilm#205]
I am using (LayoutLM):
I want to use pre trained bert-base-multilingual-cased in layout lm. But i am not getting any success. Is there any thing i need to modify, other than is-decoder=false?

**Comment by SibtainRazaJamali** [comment]
Closing the issue, since i have found solution to integrate this model.

**Comment by yuzi1949** [comment]
@SibtainRazaJamali  how do you do it ? thanks

**Comment by SibtainRazaJamali** [comment]
This model simply extends bertPreTrainedModel class
1) BertConfig.from_pretrained('bert-base-multilingual-cased',num_labels=len(labels))
2) --pre_trained_model_path= 'bert-base-multilingual-cased'
3) Initilize remaining embedding layers with kaiming-init

You might need more data.

**Comment by kwokwangliu** [comment]
@SibtainRazaJamali Is it possible to open source the code for pretraining?  I want to learn how to implement this?
Thanks

**Comment by wolfshow** [comment]
@kwokwangliu We may release the pre-training code in the future, but no bandwidth for that right now.
